### PR TITLE
repackage helm chart

### PR DESCRIPTION
**Pull request by frodopwns**
Helm chart in master is not installing CRDs properly. This PR repackages the chart.

### CODE REVIEW

File: charts/index.yaml [hunk 1]
```diff
@@ -3,19 +3,19 @@ entries:
...
-    created: "2020-05-20T10:20:46.240488-06:00"
+    created: "2020-06-01T12:41:48.174129-06:00"
     dependencies:
     - condition: azureUseMI
       name: aad-pod-identity
       repository: https://raw.githubusercontent.com/Azure/aad-pod-identity/master/charts
       version: 1.5.5
     description: Deploy components and dependencies of azure-service-operator
-    digest: f381ed2be22005190cffc85883f42a2f13512d7b789ba8a03d8ee4cf06235241
+    digest: ec95181a3e59179bfdb3bd92989613b1469104a39e6054acea90382795b1fe2d
```
Comment (WilliamMortlMicrosoft) [inline charts/index.yaml:13]: did changing the digest number fix things?

File: charts/index.yaml [hunk 2]
```diff
@@ -3,19 +3,19 @@ entries:
   azure-service-operator:
   - apiVersion: v2
     appVersion: 0.1.0
-    created: "2020-05-20T10:20:46.240488-06:00"
+    created: "2020-06-01T12:41:48.174129-06:00"
     dependencies:
     - condition: azureUseMI
       name: aad-pod-identity
       repository: https://raw.githubusercontent.com/Azure/aad-pod-identity/master/charts
```
Comment (WilliamMortlMicrosoft) [inline charts/index.yaml:10]: https://raw.githubusercontent.com/Azure/aad-pod-identity/master/charts doesnt load for me

File: charts/index.yaml (hunk 1 above)
Comment (frodopwns) [reply charts/index.yaml:13]: nope, looks like master got contaminated with templates from the helm chart PR that is currently being reviewed...added a condition around the CRD templates that prevented them from installing

File: charts/index.yaml (hunk 2 above)
Comment (frodopwns) [reply charts/index.yaml:10]: hmm wonder if it moved...

@cnadolny did the aad pod chart move? seems to be in the `aad-pod-identity` directory under charts

Comment (cnadolny) [reply charts/index.yaml:10]: This should be okay, as running the command below works:
helm repo add aad-pod-identity https://raw.githubusercontent.com/Azure/aad-pod-identity/master/charts

Comment (frodopwns) [reply charts/index.yaml:10]: Thanks! Looks good. I was able to add the repo as well using that url.

**Pull request merged**